docs/guide.md

```diff
@@ -69,6 +69,44 @@ export class somePlugin extends BasePlugin {
 }
 ```
 
+## Detach panel
+
+Panel can be detached to new window, API:
```

Comment:
If we add a description for the feature here (and i think it's optional as long as we have a description for the apis), the description should be longer.

Reply:
done